classes/Combination.php

```diff
@@ -99,6 +105,8 @@ class CombinationCore extends ObjectModel
             'low_stock_alert' => ['type' => self::TYPE_BOOL, 'shop' => true, 'validate' => 'isBool'],
             'default_on' => ['type' => self::TYPE_BOOL, 'allow_null' => true, 'shop' => true, 'validate' => 'isBool'],
             'available_date' => ['type' => self::TYPE_DATE, 'shop' => true, 'validate' => 'isDateFormat'],
+            'date_add' => ['type' => self::TYPE_DATE, 'shop' => true, 'validate' => 'isDate'],
+            'date_upd' => ['type' => self::TYPE_DATE, 'shop' => true, 'validate' => 'isDate'],
```

Comment:
Maybe use `isDateFormat`  ? I am not sure but I would feel comfortable if we really assert it's a datetime but a date

Reply:
Nope ! See https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Product.php#L477